### ISSUE TITLE
cookbook baking out italics in module titles, allow for <emphasis effect="italics"> to carry through

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#4781]
# Brief Description of Scenario
Slack thread: https://openstax.slack.com/archives/C022NETP39N/p1658333461419529 

There is a bug in baking that prevents [italics/sub/superscript markup in module titles](https://github.com/openstax/ce/issues/1782). There is currently a workaround but it requires CMs to write `&lt;i&gt;this is italics&lt;/i&gt;` rather than `<emphasis effect="italics">this is italics</emphasis>` . The assemble step works fine (converts `<emphasis>` to `<i>` in titles) but CMs were forced to write escaped HTML in the titles because the validation did not allow markup in titles.

tl;dr unescape the characters so that big homophobic HTML parser won't censor them

Step 1. Undo the [assemble fix](https://github.com/openstax/rhaptos.cnxmlutils/pull/210) so that` &lt;b&gt;` -> `<emphasis>` again
Step 2. Unbreak the content (alexa play toni braxton's unbreak my heart)

# Location(s)
- **osbooks repo:**  https://github.com/openstax/osbooks-otto-book/tree/italicsubsup
- **collection slug:** book-slug1
- **ref/branch:** italicsubsup
- **chapter/section:** 
- **surrounding text:**

**Comment by ajluk** [comment]
Hi @philschatz I have tried to remove the [title patch fix](https://github.com/openstax/rhaptos.cnxmlutils/pull/210/files) from rhaptos repo by cloning it into `enki` locally and do the changes there. I have reverted the changes made in `rhaptos/cnxmlutils/xsl/cnxml-to-html5.xsl`  from lines 222-226 and  commented out the changes made from l230. Then I tried to reassemble and bake the content form https://github.com/openstax/osbooks-otto-book/tree/italicsubsup but there were no visible changes in assembled and baked file: 
![image](https://user-images.githubusercontent.com/55203598/187939918-0c95d82c-8676-4245-83a8-797bdb65e18e.png)
![image](https://user-images.githubusercontent.com/55203598/187940292-91879daa-69b3-4d81-bbde-cea403f7ca01.png)

 Should there something more needs be done in the `enki` repo?

**Comment by philschatz** [comment]
Yeah... rhaptos.cnxmlutils is a particularly annoying thing to test inside enki. I'm planning on removing this nesting doll problem :nesting_dolls: this quarter.

In order to test the changes in enki I think you'll need to:

1. copy the cnxml-to-html5.xsl into the enki repo
1. add the following line to the Dockerfile.common in enki:

```docker
COPY ./cnxml-to-html5.xsl /workspace/enki/venv/lib/python3.8/site-packages/rhaptos/cnxmlutils/xsl/cnxml-to-html5.xsl
```

And then run `./enki ...`

**Comment by ajluk** [comment]
@lwtchu Is it possible to add some more test cases to the existing testing content for this issue (otto book: https://github.com/openstax/osbooks-otto-book/tree/italicsubsup)? Currently there is only the introduction module, it will be good to have also one more non intro module and some eoc modules, this book has default recipe/style set to chemistry, so it could be summary and some exercises module/section with class `exercises`.

**Comment by lwtchu** [comment]
@ajluk i added an appendix and another module - I'm not sure if we can do EOC? unless there's a specific recipe to test

**Comment by ajluk** [comment]
@lwtchu This book use `chemistry` recipe, becuase it was added in the `META-INF/book.xml` file (`style="chemistry"`). That's why some eoc modules supported in chemistry recipe can be added there I assume. (summary, exercises with class `exercises`), but I can prepare a recipe for testing purposes if you would like to add some different modules classes and the recipe used for baking this book can be overwritten via the `enki` pipeline script, where we can specify the style/recipe (currently the name is the same).

**Comment by lwtchu** [comment]
@ajluk i added an appendix and summary EOC - let me know if you need more? kept it super simple

**Comment by ajluk** [comment]
Screenshots for references:

1. pdf
![image](https://user-images.githubusercontent.com/55203598/188887068-7a50711c-78b4-4319-b9c3-5639a541c7bc.png)
![image](https://user-images.githubusercontent.com/55203598/188886283-a30d768f-8af9-44a4-8f01-b8831bba9d9a.png)
![image](https://user-images.githubusercontent.com/55203598/188886559-1150006e-18e1-47cd-89f7-d8eef062eecd.png)
![image](https://user-images.githubusercontent.com/55203598/188887145-8e2065d9-a4b6-46f6-ac59-5c6d8d706060.png)

3. rex-preview
![image](https://user-images.githubusercontent.com/55203598/188885791-273f9d8d-8f73-4d16-8379-c24f176fae15.png)
![image](https://user-images.githubusercontent.com/55203598/188886735-e2c1de55-20d3-40e2-ab77-61d4545c5819.png)

**Comment by therealmarv** [comment]
Short question: Nesting works without problems? Meaning e.g. a super/sub-script with italics or a superscript in a subscript?

**Comment by lwtchu** [comment]
We can try! I just updated the content with a nestedsup in the first module

**Comment by MartaSztuka** [comment]
#### PASSED
##### PDF
![Screenshot from 2022-09-13 09-56-24.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/d1a2d239-27e5-48e1-902f-19a63d02ce13)

![Screenshot from 2022-09-13 09-57-01.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/9a507d3a-bf17-4cea-adde-cd855c01a68b)

![Screenshot from 2022-09-13 09-57-19.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/3ac34298-3984-4746-a0c3-e72053174bc9)
![Screenshot from 2022-09-13 09-57-38.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/3d0fa73f-fe4b-46ba-9284-30a844845ef0)
##### REX PREVIEW
![Screenshot from 2022-09-13 10-20-58.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/3919e34e-f3dc-43d5-9025-b8a69ee9a40b)

![Screenshot from 2022-09-13 10-21-29.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/e07ca258-63cf-40f9-9986-4e5b2513e836)

**Comment by therealmarv** [comment]
Thanks for the new screenshots and updates. Looks all good for me ✅
